ts/features/pushNotifications/store/reducers/userBehaviour.ts

```diff
@@ -1,13 +1,30 @@
 import { getType } from "typesafe-actions";
 import { Action } from "../../../../store/actions/types";
-import { setEngagementScreenShown } from "../actions/userBehaviour";
+import {
+  resetNotificationBannerDismissState,
+  setEngagementScreenShown,
+  setPushNotificationBannerForceDismissed,
+  setUserDismissedNotificationsBanner
+} from "../actions/userBehaviour";
 
 export type UserBehaviourState = {
   engagementScreenShown: boolean;
+  pushNotificationsBanner: {
+    timesDismissed: number;
+    forceDismissionDate?: Date;
```

Comment:
`Date` is a type that is serialised as a string by redux-persist. What about using the timestamp (Date.getTime()) and converting this property to a number?